docs/api/structures/usb-device.md

```diff
@@ -0,0 +1,8 @@
+# USBDevice Object
+
+* `deviceId` string - Unique identifier for the device.
+* `name` string - Name of the device.
```

Comment:
- It looks like this is sourced from WebUSB's `USBDevice.productName`. Is there a reason for shortening the property name to just `name` here?

- Should this property be marked as optional? `USBDevice.productName` [is optional](https://wicg.github.io/webusb/#usbdevice).